src/Symfony/Component/Security/Http/Firewall/ContextListener.php

```diff
@@ -61,14 +61,23 @@ public function __construct(TokenStorageInterface $tokenStorage, $userProviders,
 
         $this->tokenStorage = $tokenStorage;
         $this->userProviders = $userProviders;
-        $this->contextKey = $contextKey;
```

Comment:
I think we need to deprecate the constructor argument if the class doesn't need it anymore. That can be done in another PR, could you revert this change (with the property removal)?

Reply:
It's still used for the `sessionKey`. Only the property isn't removed because it's not used.

Reply:
Correct, I've merely removed the property because the property wasn't used anymore. If desired, I can change this in a lower branch instead.

Reply:
@iltar that would be nice

Reply:
see #24329